src/main/java/htsjdk/variant/vcf/VCFHeaderLineTranslator.java

```diff
@@ -48,13 +50,19 @@
     }
 
     public static Map<String,String> parseLine(VCFHeaderVersion version, String valueLine, List<String> expectedTagOrder) {
-        return mapping.get(version).parseLine(valueLine,expectedTagOrder);
+        return mapping.get(version).parseLine(valueLine,expectedTagOrder,Collections.emptySet());
+    }
+    
+    public static Map<String,String> parseLine(VCFHeaderVersion version, String valueLine, List<String> expectedTagOrder, Set<String> optionalTags) {
+        return mapping.get(version).parseLine(valueLine,expectedTagOrder,optionalTags);
```

Comment:
```suggestion
        return mapping.get(version).parseLine(valueLine,expectedTagOrder, optionalTags);
```

Reply:
I am a bit confused - why that one extra space, while the rest of the line does not have spaces? I was just trying to be consistent here.

Reply:
I added spaces before commas in this code